nginx_keystore_requests: support lua-mendix-keystore v0.6

pommi opened the pull request:
In lua-mendix-keystore v0.4 the `/keystore/stats` format changed.

pommi committed:
nginx_keystore_requests: support lua-mendix-keystore v0.4

In lua-mendix-keystore v0.4 the `/keystore/stats` format changed.

knorrie commented:
I expected a change to a multigraph plugin and adding a second graph for the new ca stuff?

pommi committed:
nginx_keystore_requests: for loop to generate all graph items

pommi commented:
Here you go

pommi committed:
nginx_keystore_requests: support TLS Keystore CA lookups

pommi committed:
nginx_keystore_requests: support lua-mendix-keystore v0.5

In lua-mendix-keystore v0.5 TLS Keystore Certificate lookup statistics
were split into fqdn and wildcard lookups.

pommi committed:
nginx_keystore_requests: add graph for nginx shared memory object usage

pommi committed:
nginx_keystore_requests: not_found as the last item of the graph

knorrie commented:
Hooray
